pack/tdc/tdcp.json

```diff
@@ -1112,6 +1368,22 @@
         "type_code": "event",
         "xp": 3
     },
+    {
+        "code": "11080",
+        "deck_limit": 1,
+        "exceptional": true,
+        "faction_code": "mystic",
+        "illustrator": "Brian Valenzuela",
+        "name": "Eldritch Brand",
+        "pack_code": "tdcp",
+        "permanent": true,
+        "position": 80,
+        "quantity": 1,
+        "text": "Permanent. Exceptional.\nWhen you purchase this card, choose a non-signature [[Spell]] asset in your deck and record <i>&lbrack;spell's name&rbrack; was branded</i> in your campaign log. This is your branded spell. Your deck cannot include more than 1 copy of your branded spell.\n[reaction] Before you draw your opening hand: Search your deck for your branded spell and play it <i>(paying its cost)</i>.",
```

Comment:
@fspoettel Just a heads up, I used HTML entities in the text here because I was worried that `[` and `]` might be interpreted otherwise (a la `[action]` or `[intellect]`), and I don't think they can just be escaped in quite the same way in JSON.

Reply:
good thinking 👍 that should be fine.